src/ui/components/shared/SharingModal/SharingModal.tsx

```diff
@@ -12,7 +12,8 @@ import MaterialIcon from "../MaterialIcon";
 import PrivacyDropdown from "./PrivacyDropdown";
 
 function SharingModalWrapper(props: PropsFromRedux) {
-  const { recordingId } = props.modalOptions!;
+  const opts = props.modalOptions;
+  const recordingId = opts && "recordingId" in opts ? opts.recordingId : "";
```

Comment:
Typescript is ornery here. It'll complain because opts either has `recordingId` or `view` at the moment. I I think the fix is to build the action differently.